### ISSUE TITLE
io.netty.buffer bundle reference io.netty.util ,but there's not a netty util osgi bundle

### ISSUE THREAD

**Issue by gurudk** [netty/netty#3414]
in io.netty.buffer ,Import io.netty.util package , now I have two solutions:

1. embed netty-all.4.0.25.Final in io.netty.buffer, so ugly
2. create a osgi io.netty.util bundle, It;s not easy to deal with indirect dependencies

io.netty.buffer MANIFEST.MF import: 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Import-Package: io.netty.util;version="[4.0,5)",io.netty.util.concurre
 nt;version="[4.0,5)",io.netty.util.internal;version="[4.0,5)",io.nett
 y.util.internal.logging;version="[4.0,5)",org.eclipse.jetty.alpn;vers
 ion="[1,20)";resolution=optional,sun.nio.ch;resolution:=optional,org.ec
 lipse.jetty.npn;version="[1,5)";resolution=optional

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Error starting framework: org.osgi.framework.BundleException: Unresolved constraint in bundle io.netty.buffer [7]: Unable to resolve 7.0: missing requirement [7.0] osgi.wiring.package; (&(osgi.wiring.package=io.netty.util)(version>=4.0.0)(!(version>=5.0.0))) [caused by: Unable to resolve 11.0: missing requirement [11.0] osgi.wiring.package; (&(osgi.wiring.package=org.eclipse.jetty.npn)(resolution=optional)(version>=1.0.0)(!(version>=5.0.0)))]
org.osgi.framework.BundleException: Unresolved constraint in bundle io.netty.buffer [7]: Unable to resolve 7.0: missing requirement [7.0] osgi.wiring.package; (&(osgi.wiring.package=io.netty.util)(version>=4.0.0)(!(version>=5.0.0))) [caused by: Unable to resolve 11.0: missing requirement [11.0] osgi.wiring.package; (&(osgi.wiring.package=org.eclipse.jetty.npn)(resolution=optional)(version>=1.0.0)(!(version>=5.0.0)))]
	at org.apache.felix.framework.Felix.resolveBundleRevision(Felix.java:4097)
	at org.apache.felix.framework.Felix.startBundle(Felix.java:2114)
	at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:977)
	at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:964)
	at com.nbcb.notify.Main.main(Main.java:92)

**Comment by normanmaurer** [comment]
@gurudk could you please try latest 4.0.26.Final-SNAPSHOT ?

**Comment by normanmaurer** [comment]
@gurudk I think it was fixed in 4.0.26.Final, please reopen if still a problem
